package.json

```diff
@@ -4,13 +4,21 @@
   "description": "Glasgow Graduation Project - Women Shelter system",
   "main": "index.js",
   "scripts": {
-    "start": "npm --prefix server start run dev & npm --prefix client start",
-    "install": "npm --prefix server install & npm --prefix client install"
+    "client": "npm --prefix client start",
+    "server": "npm --prefix server start",
+    "dev": "NODE_ENV=development npm run client & npm --prefix server run dev",
+    "start": "npm run server",
+    "build": "cd client && npm install && npm install --only=dev --no-shrinkwrap && npm run build && cd ../",
+    "heroku-postbuild": "npm run build"
```

Comment:
maybe I'm misunderstanding the use of the Procfile (and heroku in general) - but why can't we use the prebuild/postbuild heroku hooks instead of the procfile?

Reply:
`heroku-postbuild` is run after the build, the release hook is run before every release. When you promote an app the build (slug) is copied to the next stage rather than rebuilt. Nonetheless we still need to migrate the DB, so we use the release hook.